.travis.yml

```diff
@@ -18,6 +18,12 @@ matrix:
     - rvm: 2.3
       gemfile: Gemfile.rails42
 
+jobs:
+  include:
+      rvm: 2.5
+      gemfile: Gemfile
+      script: bundle exec rake rubocop
```

Comment:
We can add the `--parallel` flag 🏎 since we will not autocorrect functionality here.